src/main/java/com/amazon/opendistroforelasticsearch/sql/plugin/SqlSettings.java

```diff
@@ -38,6 +38,7 @@
      */
     public static final String SQL_ENABLED = "opendistro.sql.enabled";
     public static final String QUERY_SLOWLOG = "opendistro.sql.query.slowlog";
+    public static final String QUERY_ANALYSIS_ENABLED = "opendistro.sql.query.analysis.enabled";
```

Comment:
Question, What is the purpose of this setting and why the we want the customer to disable this feature?

Reply:
I think this is served as a fail safe to bypass this experimental feature in case of any unexpected constraint imposed to the query.